tests/test_modeling_common.py

```diff
@@ -176,6 +176,109 @@ def test_save_load__keys_to_ignore_on_save(self):
                 for k in _keys_to_ignore_on_save:
                     self.assertNotIn(k, state_dict_saved)
 
+    def test_save_load_fast_init_from_base(self):
+        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+        base_class = MODEL_MAPPING[config.__class__]
+
+        if isinstance(base_class, tuple):
+            base_class = base_class[0]
+
+        def _mock_init_weights(self, module):
```

Comment:
We need to make the `_init_weights` function deterministic since we cannot assume that the same weights are initialized using `torch.manual_seed(...)`:

Slow init initializes all weights while fast init only some -> `torch.manual_seed(0)` doesn't lead to same init results. I don't think it's possible without making init function deterministic.

Reply:
Nevertheless, this ensures that both models have the exact same init procedure and no weight is not initialized when using "fast" init

Reply:
Oh I think I'm going to love this! :star_struck: